Client/Clique/Common/Constants/Constants.swift

```diff
@@ -6,13 +6,43 @@
 //
 
 import Foundation
+import Alamofire
+import UIKit
 
 enum Constants {
     
     enum Service {
         private static let baseURL = "https://cliquewebservice20221115180920.azurewebsites.net/api/"
         static let registerURL = baseURL.appending("Authentication/RegisterUser")
-        static let loginURL = baseURL.appending("Authentication/LoginUser")    
+        static let loginURL = baseURL.appending("Authentication/LoginUser")
+        static let profileGetUserURL = baseURL.appending("User")
+        static let profileUpdateURL = baseURL.appending("User/UpdateUserData")
+        static let passwordUpdateURL = baseURL.appending("User/UpdateUserPassword")
+        
+        static func getToken() -> String {
+            guard let token = UserStorage.token else { return "" }
+            return token
+        }
+        
+        static func authorizationHeader() -> HTTPHeaders {
+            let header: HTTPHeaders = [
+                "Authorization": "Bearer"
+                + " " + "\(getToken())",
+                "Accept": "application/json"
+            ]
+            return header
+        }
```

Comment:
The problem here is that you are doing this under a section of code called Constants. You might want to avoid that. You might want to separate this logic. Also I noticed you did not applied the formatting I suggested you to do so. This is what my suggestion would be:

```suggestion
       static func authorizationHeader() -> HTTPHeaders {
            return [
                "Authorization": "Bearer \(UserStorage.token ?? "")",
                "Accept": "application/json"
            ]
        }
```